server/src/main/java/keywhiz/service/daos/SecretSeriesDAO.java

```diff
@@ -472,9 +512,9 @@ public static class SecretSeriesDAOFactory implements DAOFactory<SecretSeriesDAO
 
   // create a new name for the deleted secret, so that deleted secret names can be reused, while
   // still having a unique constraint on the name field in the DB
-  private String transformNameForDeletion(String name) {
+  private static String transformNameForDeletion(String name) {
     long now = OffsetDateTime.now().toEpochSecond();
-    return String.format(".%s.deleted.%d.%s", name, now, UUID.randomUUID().toString());
+    return String.format(".%s.deleted.%d.%s", name, now, UUID.randomUUID());
   }
```

Comment:
Fixing a couple IDE nags:

1. This is a utility method which doesn't rely on any state from the hosting object, so I've marked it static.
2. String.format() already coerces every argument into a String by invoking toString(), so manually invoking it ourselves isn't necessary.